src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -90,6 +90,8 @@ public void drive(double x, double y, double rot) {
         double ySpeed = y*DriverConstants.maxSpeedMetersPerSecond;
         double rotSpeed = rot*DriverConstants.maxAngularSpeed;
 
+        // This doesn't account for rotSpeed < -DriverConstants.maxAngularSpeed
+        // Not sure if this is necessary, since rot will always be between -1 and 1, right?
```

Comment:
Yep good point, that was a carry over from before I made a max corrective rot speed, I'll get rid of it